tests/test_logfire.py

```diff
@@ -1548,50 +1548,6 @@ def test_readable_span_signature():
     }
 
 
-def test_tags(exporter: TestExporter) -> None:
```

Comment:
Why was this test moved? If something changed, I can't see what.

Reply:
I moved it so that all the tag tests are "near each other." If that's an issue, I can revert it.

Reply:
let's keep diffs minimal and clean, both to make this PR more readable and for future `git blame`s. i don't think it's very important for the tests to be near each other, but you can also move the new tests here.